app/src/main/java/com/kickstarter/mock/factories/PXClientFactory.kt

```diff
@@ -0,0 +1,64 @@
+package com.kickstarter.mock.factories
+
+import android.content.Context
+import com.kickstarter.libs.Build
+import com.kickstarter.libs.perimeterx.PerimeterXClient
+import okhttp3.Response
+import rx.Observable
+import rx.subjects.PublishSubject
+import java.util.HashMap
+
+class PXClientFactory private constructor() {
+
+    open class MockPXClient(build: Build) : PerimeterXClient(build) {
```

Comment:
Why did you add this class to be inside the PXClientFactory? Should it be separated?